tools/test_modules/m25100.pm

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env perl
+
+##
+## Author......: See docs/credits.txt
+## License.....: MIT
+##
+
+use strict;
+use warnings;
+
+use Digest::MD5 qw (md5 md5_hex);
+use Digest::HMAC qw (hmac hmac_hex);
+
+sub module_constraints { [[1, 256], [24, 3000], [-1, -1], [-1, -1], [-1, -1]] }
```

Comment:
Not a requirement, but I think you want to use 16 instead of 24

Reply:
Hi, 24 is the hex len, it works (same as digest len in hex for md5-96 and sha1-96)